docs/user/dashboard.asciidoc

```diff
@@ -34,54 +32,60 @@ dashboards are not visible. For more information, see <<xpack-security-authoriza
 [role="screenshot"]
 image::images/dashboard-read-only-badge.png[Example of Dashboard's read only access indicator in Kibana's header]
 
-[float]
-[[dashboard-getting-started]]
-=== Interact with dashboards
-
-When you open *Dashboard*, you're presented an overview of your dashboards. 
-If you don't have any dashboards, you can add 
-<<add-sample-data, sample data sets>>,
-which include pre-built dashboards. 
-
-Once you open a dashboard, you can filter the data
-by entering a search query, changing the time filter, or clicking 
-in the visualizations, searches, and maps. If a dashboard element has a stored query, 
-both queries are applied.
-
 --
 
 [[dashboard-create-new-dashboard]]
 == Create a dashboard
 
-To create a dashboard, you must have data indexed into {es}, an index pattern 
-to retrieve the data from {es}, and 
-visualizations, saved searches, or maps. If these don't exist, you're prompted to 
-add them as you create the dashboard.
+To create a dashboard, you must have data indexed into {es}, an index pattern
+to retrieve the data from {es}, and
+visualizations, saved searches, or maps. If these don't exist, you're prompted to
+add them as you create the dashboard, or you can add
+<<add-sample-data, sample data sets>>,
+which include pre-built dashboards.
 
 For an end-to-end example, see <<tutorial-build-dashboard, Building your own dashboard>>.
 
-. Open *Dashboard.*
-. Click *Create new dashboard.*
-. Click *Add*.
-. Use *Add panels* to add elements to the dashboard.
-+
+To begin, open *Dashboard*, then click *Create new dashboard.*
+
+[float]
+[[dashboard-add-elements]]
+=== Add elements
+
 The visualizations, saved searches, and maps
-are stored in panels that you can move and resize. A
-menu in the upper right of the panel has options for customizing
-the panel. You can add elements from 
+are stored as elements in panels that you can move and resize. The
```

Comment:
Is the sentence "The menu in the upper right" needed? Maybe its something the user can discover